dice.js

```diff
@@ -0,0 +1,9 @@
+function dice(fDice , sDice){
+    fDice= Math.floor( Math.random()*6)
+    sDice= Math.floor( Math.random()*6)
+    fDice++
+    sDice++
```

Comment:
Great idea, adding ```fDice``` and ```sDice``` by one so the range of random number will start from 1 to 6, Also there another solution to generate a random number within a range:
```Math.round(Math.random() * (max - min) + min);```